console/src/main/java/org/georchestra/console/ws/edituserdetails/EditUserDetailsFormController.java

```diff
@@ -67,6 +67,11 @@
 @SessionAttributes(types = EditUserDetailsFormBean.class)
 public class EditUserDetailsFormController {
 
+    private static final String PREAUTH_IDP = "preauth-idp";
+    private static final String PREAUTH_USER_ID = "preauth-userid";
+
```

Comment:
`preauth` prefixed headers are meant to be consumed by the georchestra gateway. Once behind the gateway, the proxified webapps should not receive them (see documentation here: https://github.com/georchestra/georchestra-gateway/blob/main/docs/pre-authentication.adoc), but receive the usual `sec-*` prefixed headers.